test/e2e/service.go

```diff
@@ -54,6 +54,9 @@ const loadBalancerLagTimeout = 2 * time.Minute
 //TODO: once support ticket 21807001 is resolved, reduce this timeout back to something reasonable
 const loadBalancerCreateTimeout = 20 * time.Minute
 
+// How many attempts to retry on staticIPName collision
+const maxAttempts = 4
```

Comment:
should use a more specific name than `maxAttempts`, which is too generic. alternatively, maybe don't need to make this a separate constant.